README.md

```diff
@@ -75,6 +76,9 @@ work the same.
 
 With the tools installed, we can begin working on our web app.
 
+
```

Comment:
Ignore changes to readme - these are notes of where I will split the content up and put into separate files under Pages. This will be deleted once all new material is edited.